modules/daggerverse/main.go

```diff
@@ -190,10 +190,6 @@ func (h *Daggerverse) BumpDaggerVersion(
 		WithExec([]string{"go", "mod", "tidy"})
 	updated := daggerio.WithDirectory("daggerverse", daggerverse.Directory("."))
 
-	api := dag.Go(daggerio.Directory("api")).Env().
-		WithExec([]string{"go", "mod", "tidy"})
-	updated = updated.WithDirectory("api", api.Directory("."))
-
```

Comment:
@jedevc this caused a failure in the daggerverse bump pr creation: https://github.com/dagger/dagger/actions/runs/13978816976/job/39139512767#step:3:4477

the `api/` subdir doesn't have it's own `go.mod` so seems safe to remove, just checking I'm not missing something